lib/reports/report.rb

```diff
@@ -30,10 +30,15 @@ def format_date(date)
   end
 
   def display_cost(cost)
```

Comment:
Does the param need to be optional if you are checking for it in the method? ie def display_cost(cost=nil)